website/public/main.css

```diff
@@ -296,15 +296,18 @@ nav.breadcrumbs ol > li {
   background-color: #ffeecc;
 }
 @media (prefers-color-scheme: dark) {
-  .comparison td.bad {
+  .comparison td.bad,
+  .comparison-mobile td.bad {
```

Comment:
Must fix: These rules only apply in dark mode, not light mode. The table is missing colors in light mode.

I think you should fix this by adding the `comparison` class to the table rather than updating all of the individual CSS selectors here.